packages/manager/src/features/Linodes/LinodeCreate/VPC/VPC.tsx

```diff
@@ -158,6 +158,11 @@ export const VPC = () => {
                     setValue('interfaces.0.subnet_id', null);
                   }
 
+                  // Clear any previously selected dual-stack values if the current vpc is not dual-stack
+                  if (isDualStackEnabled && !vpc?.ipv6) {
+                    setValue(`interfaces.0.vpc.ipv6`, undefined);
```

Comment:
I think the key `interfaces.0.vpc.ipv6` may be wrong...

```suggestion
                    setValue(`interfaces.0.ipv6`, undefined);
```

Hard to tell because IPv6 for legacy config interfaces isn't well documented, but i looks like the payload is the shape (no `vpc` object):

```json
{"purpose": "vpc", "subnet_id": 1, "ipv6": { "ranges": [{}] }}
```

Maybe this will fix the issue @dwiley-akamai  called out?

Reply:
Addressed in b05cad6